pkg/cloudprovider/servicelb.go

```diff
@@ -443,13 +435,9 @@ func (k *k3s) newDaemonSet(svc *core.Service) (*apps.DaemonSet, error) {
 	}
 	sourceRanges := strings.Join(sourceRangesSet.StringSlice(), ",")
 
-	var sysctls []core.Sysctl
 	for _, ipFamily := range svc.Spec.IPFamilies {
 		switch ipFamily {
```

Comment:
We don't need a switch anymore, or?

Reply:
I guess I could simplify to just an if ;)